test/backend/test_dynamic_shape.py

```diff
@@ -1122,6 +1124,50 @@ def test_split(self):
       o = tf_model_output["Z%i" % i]
       np.testing.assert_almost_equal(o, np.split(x, np.cumsum(split))[i])
 
+  def test_trilu(self):
+    if legacy_opset_pre_ver(14):
+      raise unittest.SkipTest("ONNX version {} doesn't support Trilu.".format(
+          defs.onnx_opset_version()))
+    shape = [2, 4, 6]
+    k = np.array(1).astype(np.int32)
```

Comment:
According to ONNX spec, input k is int64

Reply:
Updated to use np.int64.